website/pages/en/deploying/_meta.js

```diff
@@ -1,5 +1,4 @@
 export default {
-  'subgraph-studio': '',
```

Comment:
We should add a redirect from this page to `deploying-a-subgraph-to-studio`.

Reply:
fixed the title of the page on meta js file too ;)